File: TenMinsPy/day48.py
# Check if the string matches email pattern .+\@.+\..+

import re

class day48:
    def __init__(self):
        s = input("enter string\n")
        p = r".+\@.+\..+"
        if re.match(p, s):
            print('yeah man')
        else:
            print('nah man')

day48()

File: TenMinsJava/day48.java
// Check if the string matches email pattern .+\@.+\..+
package TenMinsJava;

import java.util.Scanner;

public class day48 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter a string");
        String s = scan.nextLine();
        if (s.matches(".+\\@.+\\..+")) {
            System.out.println("good man");
        }
        else {
            System.out.println("nah man");
        }
    }
}
